plugins/woocommerce-admin/client/layout/store-alerts/index.js

```diff
@@ -74,7 +74,14 @@ export class StoreAlerts extends Component {
 					isPrimary={ action.primary }
 					isSecondary={ ! action.primary }
 					href={ action.url || undefined }
-					onClick={ () => triggerNoteAction( alert.id, action.id ) }
+					onClick={ async ( event ) => {
+						const url = event.currentTarget.getAttribute( 'href' );
+						event.preventDefault();
+						await triggerNoteAction( alert.id, action.id );
+						if ( url && url !== '#' ) {
+							window.location.href = url;
+						}
```

Comment:
When testing this, I noticed that the entire page gets reloaded if the `url` is the same as the current page, or if the `url` is to another wc-admin page.

The following prevents this:

```js
if (
	url &&
	url !== '#' &&
	parseAdminUrl( url ).href !== window.location.href
) {
	navigateTo( { url } );
}
```

Using the following imports:

```js
import { navigateTo, parseAdminUrl } from '@woocommerce/navigation';
```